applications/production.py

```diff
@@ -7,11 +7,17 @@
 
     The simulations are split into two stages: showers and array.
     Shower simulations are performed with CORSIKA and array simulations \
-    with sim_telarray.
+    with sim_telarray. Note that either shower or array simulations are \
+    submitted (so typically you first run shower simulations, and then the \
+    array simulations).
 
     A configuration file is required. See tests/resources/prodConfigTest.yml \
     for an example.
 
+    The workload management system used is given in the configuration file. \
+    Allowed systems are qsub (using gridengine), condor_submit \
+    (using HTcondor), and seriell_script (running the script locally).
```

Comment:
What is seriell? Should it be serial? Or perhaps clearer, "local"?

Reply:
Good suggestion - wasn't happy with serial - use local now.